### ISSUE TITLE
CLI - Aceitar arquivo de mídia como parâmetro

### ISSUE THREAD

**Issue by leandroramos** [Hp2501/media_of_transcoder#3]
Nova funcionalidade - aceitar parâmetros na ferramenta CLI. Por exemplo: ao informar o arquivo na linha de comando, o menu da aplicação se adapta ao contexto do arquivo informado.

**Comment by Hp2501** [comment]
Opa @leandroramos você fala em ter como definir paths de origem e destino personalizados??

**Comment by leandroramos** [comment]
Sim, usando o comando (no caso, por enquanto, ./main.py, ou outro comando se for colocado no PATH) indicando o caminho do arquivo como parâmetro.
